Correct information about plugin enabling (#1760)
ansible/ansible-documentation: docs/docsite/rst/plugins/cache.rst

    [defaults]
    fact_caching = namespace.collection_name.cache_plugin_name

> To enable a custom cache plugin, save it in one of the directory sources configured in :ref:`ansible.cfg <ansible_configuration_settings>` or in a collection and enable it via it's fqcn.

Reviewer: ```suggestion
To enable a custom cache plugin, save it in one of the directory sources configured in :ref:`ansible.cfg <ansible_configuration_settings>` or in a collection and then reference it by FQCN.
```
